api.ts

```diff
@@ -2393,8 +2398,16 @@ export function makeSyncTable<
     params: ParamValues<ParamDefsT>,
     context: SyncExecutionContext,
   ): Promise<SyncFormulaResult<K, L, SchemaT>> {
-    const syncResult = (await wrappedExecute(params, context)) || {};
-    const appliedSchema = context.sync.schema;
+    let syncResult: SyncFormulaResult<K, L, SchemaDefT>;
+    try {
+      syncResult = (await wrappedExecute(params, context)) || {};
+    } catch (err: any) {
```

Comment:
since we just talked about `any` in the eng all hands, i'd love to learn - any reason not to use `unknown` here?

Reply:
The issue then is we'd need to either type `onError` as `onError(err: unknown)`, or we need some way of asserting that the object is an error. Kind of a pervasive issue throughout our codebases. Did we have a best practice as far as a path forward for errors specifically? I was hoping a future TS release would do something to help with richer typings of errors.

Reply:
Not to my knowledge :(